tests/api/v1/software/test_routes.py

```diff
@@ -1,13 +1,13 @@
-from pathlib import Path
-
 import pytest
+from anyio import Path, open_file
 
 
 @pytest.mark.asyncio
 async def test_create_software_local(async_client, test_data):
-    path = Path(__file__).resolve().parent / "software-header.swu"
-    with open(path, "rb") as file:
-        files = {"file": file}
+    resolved = await Path(__file__).resolve()
+    path = resolved.parent / "software-header.swu"
+    async with await open_file(path, "rb") as file:
+        files = {"file": await file.read()}
         response = await async_client.post(f"/api/v1/software", files=files)
```

Comment:
This should drop back a level to be outside the scope of the `async with`.  Not introduced here, but should be fixed nonetheless.